packages/app-top-navbar/components/TopNavBar/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// import { PublishIcon, ReplyIcon, Link } from '@bufferapp/components'
+
+import { calculateStyles } from '@bufferapp/components/lib/utils'
+import { black } from '@bufferapp/account-components/style/color'
+import { navbar } from '@bufferapp/account-components/style/zIndex'
+
+const style = calculateStyles({
+  default: {
+    background: black,
+    textAlign: 'center',
+    display: 'flex',
+    flex: '1',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    width: '100%',
+    height: '64px',
+    maxHeight: '64px',
+    zIndex: navbar,
+  },
+})
+
+const TopNavBar = ({ environment }) => {
+  const prefix = environment == 'development' ? 'local.' : ''
```

Comment:
```suggestion
  const prefix = environment === 'development' ? 'local.' : ''
```
Is eslint configured yet in this app? I would expect it to complain about non-strict checks.

Reply:
I'm not sure about the configuration of eslint, I got many complaints about TS issues, I guess that's for typescript right? I don't think we are using TS 🤷‍♂️